src/pages/cookie-policy/cookie-policy.constants.js

```diff
@@ -0,0 +1,113 @@
+import { styles } from './cookie-policy.styles'
+
+const sectionTitleVariant = { 
+  typography: { md: 'h4', xs: 'h5' } 
+}
+const titleVariant = {
+  typography: { md: 'h5', xs: 'h5' }
+}
+const titleWithDotVariant = {
+  typography: { xlg: 'subtitle1' }
+}
+const descriptionVariant = {
+  typography: { md: 'body1', xs: 'body2' }
+}
+
+const titleWithDotStyles = {
+  title: { 
+    ...styles.titleWithDot 
+  },
+  wrapper: { 
+    ...styles.wrapper
+  }
+}
+
+export const cookieItemsData = [
+  {
+    id: 1,
+    title: 'cookiePolicyPage.cookiePolicy.title',
+    description: 'cookiePolicyPage.cookiePolicy.description',
+    titleStyles: sectionTitleVariant,
```

Comment:
These are the cookie items, you should move style from here.

Reply:
fixed